osf/models/metaschema.py

```diff
@@ -59,7 +59,7 @@ def get_latest_versions(self, request=None):
         :param request: the request object needed for waffling
         :return: queryset
         """
-        queryset = self.filter(visible=True).order_by('name', '-schema_version').distinct('name')
+        queryset = self.order_by('name', '-schema_version').distinct('name')
```

Comment:
There are three places where this is used.

1. api.schemas.views:RegistrationSchemaList::get_default_queryset()
2. website.project.views.drafts::get_metaschemas()
3. api.providers.views:RegistrationProviderSchemaList::get_default_queryset()

The first two expect only visible items to return. The third expects to get them regardless of visibility. So the thing that makes the most sense to me is to have a parameter to show invisible that defaults to `False`, and set it in `api.providers.views:RegistrationProviderSchemaList::get_default_queryset()` to `True`.

The reason for the difference is that the first two are related to the traditional method of making a registration, which is to go through a node's registration tab and make a registration from there. That defaults to OSF as the provider, and we haven't updated that to use the new `api.providers.views:RegistrationProviderSchemaList`. Therefore, it needs to have all of the latest, active, visible schemas and none of the rest. 

Whereas anything using the new `api.providers.views:RegistrationProviderSchemaList` needs to have whatever schemas are explicitly defined in the admin app for use. So, when we get provider Foo with schema Bar that is only used by Foo, we set that to be invisible to OSF, but we add them with the admin app on that provider. It shows in the new view but not the older views. Does that make sense?